[Permissions Chip article] Chrome 97 → 98 (#2075)
GoogleChrome/developer.chrome.com: site/en/blog/permissions-chip/index.md

Chrome's telemetry data proves that a lot of permission prompts are ignored. You can explore the
[notification permission data](https://developers.google.com/web/updates/2020/02/notification-permission-data-in-crux)
in the Chrome UX report yourself. For now, consider the table below that shows

> how Windows users reacted to the notification prompt on sites in an accumulated way:

Reviewer: If removing the geolocation metrics, we should still be able to say we see similar dismiss / ignore rates for geolocation - being the second most requested permission type.

Author: Added.

> how Windows users reacted to the notification prompt on sites in an accumulated way:

Author: ```suggestion
how Windows users reacted to the notification prompt on sites in an accumulated way, well noting
that geolocation prompts saw a similar dismiss or ignore behavior.
```

Chrome's telemetry data proves that a lot of permission prompts are ignored. You can explore the
[notification permission data](https://developers.google.com/web/updates/2020/02/notification-permission-data-in-crux)
in the Chrome UX report yourself. For now, consider the table below that shows

> how Windows users reacted to the notification prompt on sites in an accumulated way, well noting

Reviewer 2: ```suggestion
how Windows users reacted to the notification prompt on sites in an accumulated way, while noting
```

Given an ignore/dismiss rate of approximately 85%, and especially given how much the prompt stands out and
insists on users making a decision immediately, there is a conflict between the level of urgency
assumed by the browser versus the user's preference for waiting to make a decision. This creates the
perception that it's "annoying" for a site to ask for a permission, apart from potential extra

> things users need to react to like cookie consent banners, newsletter signups, etc.

Reviewer 2: I don't understand what the added text is trying to tell me. Please explain in more words so I can make a suggestion.

Author: When you land on a page you have to deal with the notifications prompt, then with the cookie banner, then the newsletter signup,… It's many things for just reading an article.

Reviewer 2: I think I understand. Permissions get lost because of the general site noise.

Author: Lost as in clicked away without even being read and thus contributing to the general perception of the web being annoying.

Reviewer 2: I just made a suggestion.

> perception that it's "annoying" for a site to ask for a permission, apart from potential extra
> things users need to react to like cookie consent banners, newsletter signups, etc.

Reviewer 2: ```suggestion
perception that it's "annoying" for a site to ask for a permission because it will be lost in the potential extra
things users need to react to like cookie consent banners, newsletter signups, etc.
```

## New design

From Chrome 98 on, we therefore introduced an animated chip UI which appears next to the lock
whenever a permission is requested. This consists of an icon and label describing the permission being
requested. Our aim was to improve the experience of web browsing while avoiding permission requests
that are generally unnecessary for the vast majority of users and frequently ignored or dismissed.

> {% Video src="video/8WbTDNrhLsU0El80frMBGE4eMCD3/4rH9koYThnPvEgLu4mF6.mp4", autoplay=true, muted=true, playsinline=true, loop=true%}

Reviewer 2: If the alt property was wrong, please replace it rather than removing it.

Author: This replaces an image with a video. Videos don't have `alt`.

## Outlook and conclusions

> There are plans for further UI and UX improvements the Chrome team is working on
> and the team is also investigating potentially more aggressive auto-blocking of permissions

Reviewer 2: ```suggestion
There are plans for further UI and UX improvements. The Chrome team is already working on
them and investigating potentially more aggressive auto-blocking of permissions
```
